### DISCUSSION TITLE
Undo after bought

### DISCUSSION THREAD

**Discussion by comonier** [QuickShop-Community/QuickShop-Hikari#1775] [Ideas]
I didn't want to compare that, I regret it! 

I will give /qs undo and get my money back, I don't mind returning what was purchased.

so basically the system removes the item from the player, and returns the money

If it is a player store and not a server store, the system returns to the player store [chest] the amount of stock that was there before the purchase.

**Comment by JosTheDude** [comment]
Not a bad idea, I would say the difficulty might be in tracking the recent purchases and restoring items / managing the refund system without paving the way for potential exploits

**Comment by creatorfromhell** [comment]
I think this suggestion most likely won't be implemented because there's a lot of potential for abuse/exploiting that introducing this may lead to.
